vars/kibanaPipeline.groovy

```diff
@@ -262,10 +262,13 @@ def buildXpack() {
 }
 
 def runErrorReporter() {
+  def status = buildUtils.getBuildStatus()
+  def dryRun = status != "ABORTED" ? "" : "--no-github-update"
```

Comment:
I am not seeing `no-github-check` do you mean `dry-run`?

Reply:
nevermind, I see that was added in https://github.com/elastic/kibana/pull/52314 which my local checkout didn't include.